### ISSUE TITLE
New chat replies get routed to new inaccessible window.

### ISSUE THREAD

**Issue by stefan-pdx** [beechannels/beechat#12]
Using the 855c6a4 release, I've noticed a strange behavior with a majority of users where:

"User A" (User.A@mydomain) sends a message to "User B" (User.B@mydomain). "User B" window pops up on A's screen and "User A" window pops up on B's display with the sent message. When B replies to the message, a new window is created titled "User.B@myd..." on A's display that is not selectable/expandable. A can still send messages to B, but A cannot receive messages from B.

I'm starting to poke around in the codebase a bit, but I'm somewhat unfamiliar with the project. Please let me know if this is a potential mis-configuration.

**Comment by stefan-pdx** [comment]
FWIW, I'm using the external auth method as described in the documentation to integrate beechat with Elgg.

**Comment by stefan-pdx** [comment]
Yeah - so it appears that this only happens when the user name has caps in it. In the previous scenario, where A message B, if B's name was John.Smith@mydomain, the id for the chat container would be john.smith@mydomain. However, when user B replies back to A, a second chat window is opened up with a corresponding id of John.Smith@mydomain. Looks like that the corresponding fix has to to with the improper tagging/comparison of the chat windows. I can take a look at it this coming weekend.

**Comment by dpasqualin** [comment]
You've never fixed this issue, have you? :(
